From 13aaf3bbb3d38a719917b3679054fa02a5ee18bb Mon Sep 17 00:00:00 2001
From: techyminati <techyminati@outlook.com>
Date: Sun, 8 Mar 2026 12:16:02 +0530
Subject: [PATCH 5/7] kernel_configs: Lower 4.19 requirements to 4.19.127 *
 Alot of mtk devices use 4.19-r kernels, which were shipped with 4.19.127,
 since r/4.19 does not exist on 16qpr2, lets lower min req for 4.19-s

Change-Id: I0fb9bc1de56297b49b6fa6cbd5ec97e2e5e5431b
Signed-off-by: techyminati <techyminati@outlook.com>
---
 s/android-4.19/android-base-conditional.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/s/android-4.19/android-base-conditional.xml b/s/android-4.19/android-base-conditional.xml
index 8aab2a0..24e2b47 100644
--- a/s/android-4.19/android-base-conditional.xml
+++ b/s/android-4.19/android-base-conditional.xml
@@ -1,4 +1,4 @@
-<kernel minlts="4.19.191" />
+<kernel minlts="4.19.127" />
 
 <!-- KEEP ALPHABETICALLY SORTED -->
 <!-- ARM base requirements -->
-- 
2.46.0

